php-frontend/src/main/java/org/sonar/php/parser/TreeFactory.java

```diff
@@ -0,0 +1,1597 @@
+import org.sonar.php.tree.impl.statement.DoWhileStatementTreeImpl;
+import org.sonar.php.tree.impl.statement.ElseClauseTreeImpl;
+import org.sonar.php.tree.impl.statement.ElseifClauseTreeImpl;
+import org.sonar.php.tree.impl.statement.EmptyStatementImpl;
+import org.sonar.php.tree.impl.statement.ExpressionStatementTreeImpl;
+import org.sonar.php.tree.impl.statement.ForEachStatementTreeImpl;
+import org.sonar.php.tree.impl.statement.ForEachStatementTreeImpl.ForEachStatementHeader;
+import org.sonar.php.tree.impl.statement.ForStatementTreeImpl;
+import org.sonar.php.tree.impl.statement.ForStatementTreeImpl.ForStatementHeader;
+import org.sonar.php.tree.impl.statement.GlobalStatementTreeImpl;
+import org.sonar.php.tree.impl.statement.GotoStatementTreeImpl;
+import org.sonar.php.tree.impl.statement.IfStatementTreeImpl;
+import org.sonar.php.tree.impl.statement.InlineHTMLTreeImpl;
+import org.sonar.php.tree.impl.statement.LabelTreeImpl;
+import org.sonar.php.tree.impl.statement.NamespaceStatementTreeImpl;
+import org.sonar.php.tree.impl.statement.ReturnStatementTreeImpl;
+import org.sonar.php.tree.impl.statement.StaticStatementTreeImpl;
+import org.sonar.php.tree.impl.statement.SwitchStatementTreeImpl;
+import org.sonar.php.tree.impl.statement.ThrowStatementTreeImpl;
+import org.sonar.php.tree.impl.statement.TryStatementImpl;
+import org.sonar.php.tree.impl.statement.UnsetVariableStatementTreeImpl;
+import org.sonar.php.tree.impl.statement.UseStatementTreeImpl;
+import org.sonar.php.tree.impl.statement.VariableDeclarationTreeImpl;
+import org.sonar.php.tree.impl.statement.WhileStatementTreeImpl;
+import org.sonar.php.tree.impl.statement.YieldStatementTreeImpl;
+import org.sonar.plugins.php.api.tree.CompilationUnitTree;
+import org.sonar.plugins.php.api.tree.ScriptTree;
+import org.sonar.plugins.php.api.tree.Tree;
+import org.sonar.plugins.php.api.tree.Tree.Kind;
+import org.sonar.plugins.php.api.tree.declaration.ClassDeclarationTree;
+import org.sonar.plugins.php.api.tree.declaration.ClassMemberTree;
+import org.sonar.plugins.php.api.tree.declaration.ClassPropertyDeclarationTree;
+import org.sonar.plugins.php.api.tree.declaration.ConstantDeclarationTree;
+import org.sonar.plugins.php.api.tree.declaration.FunctionDeclarationTree;
+import org.sonar.plugins.php.api.tree.declaration.MethodDeclarationTree;
+import org.sonar.plugins.php.api.tree.declaration.NamespaceNameTree;
+import org.sonar.plugins.php.api.tree.declaration.ParameterListTree;
+import org.sonar.plugins.php.api.tree.declaration.ParameterTree;
+import org.sonar.plugins.php.api.tree.declaration.VariableDeclarationTree;
+import org.sonar.plugins.php.api.tree.expression.ArrayAccessTree;
+import org.sonar.plugins.php.api.tree.expression.ArrayInitializerTree;
+import org.sonar.plugins.php.api.tree.expression.ArrayPairTree;
+import org.sonar.plugins.php.api.tree.expression.AssignmentExpressionTree;
+import org.sonar.plugins.php.api.tree.expression.CastExpressionTree;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.plugins.php.api.tree.expression.CastExpressionTree'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)